src/player.cpp

```diff
@@ -10421,6 +10422,10 @@ bool player::can_wield( const item &it, bool interactive ) const
                 add_msg( m_info, _("You are too weak to wield %s with only one arm."),
                          it.tname().c_str() );
             }
+            if( worn_with_flag("RESTRICT_HANDS") ) {
+                add_msg( m_info, _("You are wearing something that hinders the use of both hands."),
```

Comment:
You should probably replace `something` with `%s` to actually include the name of the item.